.circleci/config.yml

```diff
@@ -109,6 +109,6 @@ workflows:
                 - master
       - codacy/tag_version:
           name: tag_version
-          context: CodacyDocker
+          context: CodacyAWS
           requires:
-            - publish_dockerhub
+            - codacy/publish_docker
```

Comment:
As the above action has a name set `name: publish_dockerhub`, I have a feeling we want to leave it as the specific defined name, or remove the name property of the `codacy/publish_docker` stanza.

Reply:
thank you! I changed it